be/src/Project/Services/DownloadCenter/DownloadCenter.Api/v1/Controllers/DownloadCenterController.cs

```diff
@@ -0,0 +1,359 @@
+﻿using DownloadCenter.Api.Services.DownloadCenter;
+using DownloadCenter.Api.ServicesModels.DownloadCenter;
+using DownloadCenter.Api.v1.RequestModel;
+using DownloadCenter.Api.v1.ResponseModel;
+using DownloadCenter.Api.Validators.DownloadCenter;
+using Foundation.Account.Definitions;
+using Foundation.ApiExtensions.Controllers;
+using Foundation.Extensions.Definitions;
+using Foundation.Extensions.Exceptions;
+using Foundation.Extensions.ExtensionMethods;
+using Foundation.Extensions.Helpers;
+using Foundation.GenericRepository.Paginations;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Net;
+using System.Security.Claims;
+using System.Threading.Tasks;
+
+namespace DownloadCenter.Api.v1.Controllers
+{
+    [ApiVersion("1.0")]
+    [Route("api/v{version:apiVersion}/[controller]")]
+    [Authorize]
+    [ApiController]
+    public class DownloadCenterController : BaseApiController
+    {
+        private readonly IDownloadCenterService downloadCenterService;
+
+        public DownloadCenterController(
+            IDownloadCenterService downloadCenterService)
+        {
+            this.downloadCenterService = downloadCenterService;
+        }
+
+        /// <summary>
+        /// Get download center category by id.
```

Comment:
**suggestion:** let's fix the comments in the controller. They are no longer about category but download center and files.